Source/Scene/Cesium3DTile.js

```diff
@@ -0,0 +1,1018 @@
+        TileBoundingRegion,
+        TileBoundingSphere,
+        TileOrientedBoundingBox) {
+    'use strict';
+
+    /**
+     * A tile in a 3D Tiles tileset.  When a tile is first created, its content is not loaded;
+     * the content is loaded on-demand when needed based on the view.
+     * <p>
+     * Do not construct this directly, instead access tiles through {@link Cesium3DTileset#tileVisible}.
+     * </p>
+     *
+     * @alias Cesium3DTile
+     * @constructor
+     */
+    function Cesium3DTile(tileset, basePath, header, parent) {
+        this._tileset = tileset;
+        this._header = header;
+        var contentHeader = header.content;
+
+        /**
+         * The local transform of this tile
+         * @type {Matrix4}
+         */
+        this.transform = defined(header.transform) ? Matrix4.unpack(header.transform) : Matrix4.clone(Matrix4.IDENTITY);
+
+        var parentTransform = defined(parent) ? parent.computedTransform : tileset.modelMatrix;
+        var computedTransform = Matrix4.multiply(parentTransform, this.transform, new Matrix4());
+
+        /**
+         * The final computed transform of this tile
+         * @type {Matrix4}
+         * @readonly
+         */
+        this.computedTransform = computedTransform;
+
+        this._boundingVolume = this.createBoundingVolume(header.boundingVolume, computedTransform);
+        this._boundingVolume2D = undefined;
+
+        var contentBoundingVolume;
+
+        if (defined(contentHeader) && defined(contentHeader.boundingVolume)) {
+            // Non-leaf tiles may have a content bounding-volume, which is a tight-fit bounding volume
+            // around only the models in the tile.  This box is useful for culling for rendering,
+            // but not for culling for traversing the tree since it does not guarantee spatial coherence, i.e.,
+            // since it only bounds models in the tile, not the entire tile, children may be
+            // outside of this box.
+            contentBoundingVolume = this.createBoundingVolume(contentHeader.boundingVolume, computedTransform);
+        }
+        this._contentBoundingVolume = contentBoundingVolume;
+        this._contentBoundingVolume2D = undefined;
+
+        var viewerRequestVolume;
+        if (defined(header.viewerRequestVolume)) {
+            viewerRequestVolume = this.createBoundingVolume(header.viewerRequestVolume, computedTransform);
+        }
+        this._viewerRequestVolume = viewerRequestVolume;
+
+        /**
+         * The error, in meters, introduced if this tile is rendered and its children are not.
+         * This is used to compute screen space error, i.e., the error measured in pixels.
+         *
+         * @type {Number}
+         * @readonly
+         */
+        this.geometricError = header.geometricError;
+
+        var refine;
+        if (defined(header.refine)) {
+            refine = (header.refine === 'replace') ? Cesium3DTileRefine.REPLACE : Cesium3DTileRefine.ADD;
```

Comment:
Just for my own education, is `'replace'` and enum value in the spec or a free form string?  Normally I would expect all caps for constant known string values.